### ISSUE TITLE
Add licence to submission page

### ISSUE THREAD

**Issue by crazy4pi314** [unitaryfund/metriq-app#270]
Might be good to collect the license on code if there is a linked repo at the submission time. Might be possible to do automatically. Just ask for URL to license file.

**Comment by willzeng** [comment]
Was this closed by https://github.com/unitaryfund/metriq-app/commit/367e55a52325b6c28aa297343ccbb5f85f921dce?

**Comment by WrathfulSpatula** [comment]
@willzeng No, it was not closed by that commit. I think Sarah is referring, for example, to the license of a submitted GitHub repo, referring to the repository license, not the Metriq "submission" data license.

**Comment by willzeng** [comment]
Ah I see. Yes that is different. 

While I can see the benefit, I'd propose that for now we keep the submission process as simple and short as possible to encourage contributions.

**Comment by WrathfulSpatula** [comment]
No harm in keeping the issue open, in our backlog. We could make a "backlog" label to aid filtering, but ~40 open issues isn't too much noise for me, yet.

**Comment by crazy4pi314** [comment]
Yeah, @WrathfulSpatula thanks for expanding on what I meant here :) Ideally this is something that could be scraped from the linked repo, and if not would be added to the submission manually. Not a high priority, but I can see being able to filter submissions by license type a very useful thing in the future.
